shared/desktop/renderer/index.js

```diff
@@ -53,6 +53,42 @@ function setupAvatar() {
   initAvatarLoad(loadUserImageMap, loadTeamImageMap)
 }
 
+var timeoutID
+
+// Consider us inactive after a minute of no input
+// for the purpose of marking chat messages read
+function setTimeoutListeners() {
```

Comment:
ok got it on the 5 mins.

RE capture: not following - we need to capture the mouse? It worked without it so far.